### PR TITLE
Add apphub.admin role for the organization in the test environment

### PR DESCRIPTION
**Pull request by krishnangopal1810**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:REPLACEME

```

### CODE REVIEW

File: .ci/infra/terraform/main.tf [hunk 1]
```diff
@@ -135,6 +135,12 @@ resource "google_organization_iam_member" "sa_storage_admin" {
   member = google_service_account.sa.member
 }
 
+resource "google_organization_iam_member" "apphub_admin" {
+  billing_account_id = data.google_organization.org.org_id
```
Comment (roaks3) [inline .ci/infra/terraform/main.tf:139]: Should be `org_id`

**Review by roaks3 — CHANGES_REQUESTED** [review]
LGTM except small typo.

I've already gone ahead and granted this permission for all 3 test runners in our test org

**Review by roaks3 — APPROVED** [review]
Approved (and changes have been made to the test env), but I'm going to keep this unmerged until everyone is fully settled on this approach. It's possible we want to revisit this later and use project-level permissions in some way.

**Comment by roaks3** [comment]
@krishnangopal1810 circling back on this: do you see any reason to change the strategy here in the short-term. I believe we should probably go ahead and merge this unless we expect it to change soon.

**Comment by krishnangopal1810** [comment]
> @krishnangopal1810 circling back on this: do you see any reason to change the strategy here in the short-term. I believe we should probably go ahead and merge this unless we expect it to change soon.

I don't see a reason to change the strategy here. I agree we should merge this change.

**Review by roaks3 — APPROVED** [review]

**Pull request merged**